CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
 # Changelog
 
 ## [MAJOR.MINOR.PATCH] - YYYY-MM-DD
-+ add `webhooks.containerPort` value
+
+## aiven-operator-v0.5.0 - 2022-11-28
+
+- Add ability to link resources through the references
```

Comment:
i would just go with "sync with aiven-operator v0.5.0" ( and add the new configuration options here )